### ISSUE TITLE
Document the working functionality

### ISSUE THREAD

**Issue by ForNeVeR** [ForNeVeR/wpf-math#30]
We need the list of LaTeX commands we support. Put it somewhere into the documentation and make it available for the end-user.

**Comment by alexreg** [comment]
Good idea. Something like the README for https://github.com/opencollab/jlatexmath would be nice.

**Comment by ForNeVeR** [comment]
Ping from #267. We've done a good work on colors and matrices, but definitely should add more documentation on the supported symbols and predefined formulas.
